app/components/listing/NotesList.tsx

```diff
@@ -13,12 +13,3 @@ const NotesList = ({ notes }: { notes: Note[] }) => (
     ))}
   </ul>
 );
-
-export const Notes = ({ id, notes }: { id?: string; notes: Note[] }) => (
-  <section className="service--section" id={id} data-cy="notes-section">
```

Comment:
removed this because of site's overuse of <header> and hardcoded <h4>. Pared this down to just NotesList to add to the reusable ListingInfoSection instead.